core/services/bulletprooftxmanager/eth_broadcaster.go

```diff
@@ -268,6 +277,16 @@ func (eb *ethBroadcaster) handleInProgressEthTx(etx models.EthTx, attempt models
 		return errors.Errorf("invariant violation: expected transaction %v to be in_progress, it was %s", etx.ID, etx.State)
 	}
 
+	if eb.config.OptimismGasFees() {
+		callMsg := ethereum.CallMsg{To: &etx.ToAddress, From: etx.FromAddress, Data: etx.EncodedPayload}
+		gasLimit, err := eb.ethClient.EstimateGas(context.TODO(), callMsg)
+		if err != nil {
+			return err
+		}
+		etx.GasLimit = gasLimit
+		attempt.GasPrice = *utils.NewBig(big.NewInt(1))
```

Comment:
Does it have to be set to 1? Is this what Optimism requires?

Reply:
Yes.

Reply:
@samsondav & @j16r It should be 1 gwei, or 1000000000 wei. Can you please confirm that this is correct, or it needs an update?

Reply:
Resolved: [Optimism gas price should be 1 GWei #4285](https://github.com/smartcontractkit/chainlink/pull/4285)